### ISSUE TITLE
Remote host

### ISSUE THREAD

**Issue by dag08** [bjarque/t265_UE#9]
Hi, I want not just to have the localhost for the sending machine. The background is I want to connect the T265 to a laptop and then sending over local network to the receiving machine via WiFi so I do not have a cables between.
Do you have any clue how to do it?

**Comment by bjarque** [comment]
Yes, you can add the IP(s) it should be sending to in the commandline. 
"py ./tracker.py 192.168.1.x"
